app/api/sentiment.py

```
@@ -0,0 +1,1245 @@
+            "category": result.category.value,
+            "timestamp": result.timestamp,
+            "processed_at": datetime.utcnow(),
+            "model_version": result.model_version,
+            "processing_time_ms": result.processing_time_ms,
+            "emotion_scores": result.emotion_scores,
+            "aspect_sentiments": result.aspect_sentiments,
+            "keyword_contributions": result.keyword_contributions,
+            "user_id": str(auth["_id"]) if isinstance(auth, dict) and auth else None,
+        }
+
+        db_result = await db.sentiment_analyses.insert_one(doc_data)
```

Ruff (0.13.2)

Local variable `db_result` is assigned to but never used

Remove assignment to unused variable `db_result`

(F841)